agenthub/self_discover_agent/prompt.py

```diff
@@ -0,0 +1,214 @@
+from opendevin.core.message import Message, TextContent
+
+from .agent_state_machine import SelfDiscoverState
+from .reasoning_action import REASONING_MODULE_LIST, ReasoningAction
+
+TASK_KEY = 'task'
+
+ROLE_DESCRIPTION = """You are a seasoned manager at a Fortune 500 company with a team of capable software engineers.
+Your responsibility is to plan the given task for your team. You must only plan and must not code yourself."""
+
+INTERACTION_SKILL = """You are working with a curious user that needs your help. You can directly interact with the user by replying with
+<execute_ask> and </execute_ask>. For example, <execute_ask> Are my assumptions correct? </execute_ask>.
+"""
+
+BROWSING_SKILL = """You can also browse the Internet with <execute_browse> and </execute_browse>.
+For example, <execute_browse> Tell me the usa's president using google search </execute_browse>.
+Or <execute_browse> Tell me what is in http://example.com </execute_browse>.
+"""
+
+SYSTEM_SUFFIX = """Let the following principles guide your response:
+- Before responding read all information carefully.
+- Take time to think.
```

Comment:
Just commenting: I wonder whether this instruction was _ever_ followed. 😂

Reply:
Yeah good question. I don't have any data to answer...